src/ci/bin/testSbt.sh

```diff
@@ -27,7 +27,7 @@ esac
 export CROMWELL_SBT_TEST_EXCLUDE_TAGS
 export CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR
 
-sbt -Dakka.test.timefactor=${CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR} -Dbackend.providers.Local.config.filesystems.local.localization.0=copy coverage test
+sbt -Dakka.test.timefactor=${CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR} -Dbackend.providers.Local.config.filesystems.local.localization.0=copy coverage "${BUILD_SUB_TYPE}/test"
```

Comment:
Wait.... will this mean that we don't get a single composite "all the tests' coverage" value any more?

Reply:
Possibly related screenshot?

![Screen Shot 2019-09-09 at 11 21 50 AM](https://user-images.githubusercontent.com/13006282/64543897-09d02580-d2f4-11e9-94a0-87247c162f1f.png)